src/komponenter/utvandret/kontakt-meg.tsx

```diff
@@ -3,35 +3,73 @@ import Panel from 'nav-frontend-paneler';
 import { Systemtittel, Normaltekst } from 'nav-frontend-typografi';
 import { Hovedknapp } from 'nav-frontend-knapper';
 import '../oppholdstillatelse/kontakt-meg-melding.less';
+import { getHeaders, MED_CREDENTIALS } from '../../ducks/api';
+import { uniLogger } from '../../metrikker/uni-logger';
+import OppgaveOpprettet from './oppgave-opprettet';
 
 const KontaktMeg = () => {
 
-    return (
-        <Panel border>
-            <Systemtittel className="avbryt-modal__beskrivelse blokk-m">
-                En veileder må hjelpe deg slik at du blir registrert
-            </Systemtittel>
-            <Normaltekst className="blokk-s">
-                Du står registrert som utvandret i våre systemer.<br />
-                Dette gjør at du ikke kan registrere deg som arbeidssøker på nett.
-            </Normaltekst>
-            <Normaltekst className="blokk-m">
-                Kontakt oss, så hjelper vi deg videre.
-            </Normaltekst>
-            <div className="blokk-s">
-                <Hovedknapp className="avbryt-modal__knapp blokk-s" id="confirmKnapp" onClick={() => {}}>
-                    Ta kontakt / Contact us
-                </Hovedknapp>
-            </div>
-            <Normaltekst className="blokk-s">
-                You're listed as emigrated in our systems.<br />
-                This means that you cannot register as a jobseeker online.
-            </Normaltekst>
-            <Normaltekst className="blokk-m">
-                Please contact us for help with this.
-            </Normaltekst>
-        </Panel>
-    );
-};
+    interface Oppgave {
+        id: number;
+        tildeltEnhetsnr: string;
+        oppgaveType: string;
+    }
+
+    const [oppgave, setOppgave] = React.useState<Oppgave>({
+        id: -1,
+        tildeltEnhetsnr: '',
+        oppgaveType: ''
+    });
+
+    const opprettOppgave = async (url) => {
```

Comment:
Jeg har prøvd å gjøre kompenten mindre avhengig av den asynkrone redux-riggen + hooks ♥️. Hva synes dere om dette oppsettet med `fetch` som en del av komponenten?

Reply:
Jeg er veldig glad i løsrivelse og friere komponenter :-)
Antar det er polyfills e.l. her for `fetch` for nettlesere som ikke støtter dette? https://caniuse.com/#feat=fetch

Reply:
Virker veldig bra. Nå er det lett å se hvordan kallet gjøres i motsetning til å måtte klikke seg gjennom hele redux.

Reply:
Kjenner ikke til om det ligger noen polyfills for fetch i denne appen, men det er fetch som har vært brukt i bakgrunnen her hele tiden så antar alt er good 🤔